package/kedro_viz/server.py

```diff
@@ -229,8 +244,29 @@ def _call_viz(host=None, port=None, browser=None, load_file=None, save_file=None
                 sys.exit(1)
     else:
         try:
+            # Kedro 0.15.0+
+            from kedro.context import KedroContextError
+
+            context = get_project_context()
+            try:
+                # Kedro 0.15.2+
+                pipeline = context._get_pipeline(  # pylint: disable=protected-access
+                    name=pipeline_name
+                )
```

Comment:
This in turn can raise `KedroContextError` if the pipeline `pipeline_name` doesn't exist. But this is caught further down and an unrelated messages is printed ("could not find a Kedro project root").

Reply:
Thanks! Makes sense, I will raise a different message for this case. `try/except` is getting quite messy on this function to do backward-compatiblity....